src/profile.c

```diff
@@ -44,22 +65,38 @@ static DWORD WINAPI profile_bt( LPVOID lparam )
                 fputs("failed to suspend main thread. aborting profiling.",stderr);
                 break;
             }
-            CONTEXT ctxThread;
+
+            LPTOP_LEVEL_EXCEPTION_FILTER prevHandler = SetUnhandledExceptionFilter(profile_exception_handler);
+
+            CONTEXT ctxThread __attribute__((aligned(4)));
```

Comment:
ah, whoops. didn't mean to include this. but, FYI: there is a bunch of verbiage in every discussion I read saying that `CONTEXT` must be DWORD-aligned (probably is already by default with gcc?)